source/language/pulses.rst

```diff
@@ -16,10 +16,7 @@ instructions to the underlying microcoded
 :cite:`wilkesBestWayDesign1989` stimulus programs emitted by
 the controllers to implement each operation. In OpenQASM we expose
 access to this level of control with pulse-level definitions of gates
-and measurement with user-selectable pulse grammar. A future document
-will define a textualized representation of one such grammar, OpenPulse.
-Here we restrict ourselves to defining the necessary interfaces within
-OpenQASM to these pulse-level definitions of gates and measurement.
+and measurement using a `text representation of OpenPulse <openpulse.html>`_ 
```

Comment:
Having a user-selectable pulse grammar is important to allow systems with very different control requirements (e.g. trapped ions) to plug in an appropriate pulse description.

Reply:
Will add back in